lxc/project.go

```diff
@@ -121,15 +141,18 @@ func (c *cmdProjectCreate) run(cmd *cobra.Command, args []string) error {
 	// Create the project
 	project := api.ProjectsPost{}
 	project.Name = resource.name
+	project.ProjectPut = stdinData
+
+	if project.Config == nil {
```

Comment:
Is there any case where `project.Config` is not nil?

Reply:
Yes, when using YAML as an input for example (above, in the `if !termios.IsTerminal(getStdinFd())` block). This will populate `stdinData` which will be added to project.ProjectPut (therefore, indirectly to `project.Config`)... This is when you do `lxc project create --config k1=v1 --config k2=v2 ...` (so not using YAML) that we enter this `if project.Config == nil` block

Reply:
I see, thanks